zng/docs/spec.md

```diff
@@ -386,6 +412,7 @@ is interpreted as follows:
 | `bstring`  | variable |  UTF-8 byte sequence with `\x` escapes           |
 | `enum `    | variable |  UTF-8 bytes of enum string                      |
 | `ip`       | 4 or 16  |  4 or 16 bytes of IP address                     |
+| `port`     | XXX      |  XXX - to be filled in during PR review          |
```

Comment:
```suggestion
| `port`     | 2      |  2 byte unsigned int          |
```

Reply:
Nit: `s/2 byte unsigned int/unsigned int of length 2/` for consistency with other rows.